packages/cli/src/lib/setup/setupInstall.ts

```diff
@@ -65,6 +65,25 @@ interface CreateInstanceOptions {
     port?: number;
 }
 
+function deleteFoldersRecursive(path: string): void {
```

Comment:
No need to define this, just use https://github.com/ioBroker/ioBroker.js-controller/blob/2d349c12f7c4d538eb0a33731724d858205f6790/packages/cli/src/lib/setup/setupBackup.ts#L193

Reply:
Done